### DISCUSSION TITLE
Can I define multiple methods in server/API/user. ts? For example, defining add and update methods?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#23670] [Questions]
### Describe the feature

Can I define multiple methods in server/API/user. ts? For example, defining add and update methods?

Must there be a method to define a file?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by warflash** [comment]
See the docs file system based http method routing here: https://nuxt.com/docs/guide/directory-structure/server#matching-http-method
